lib/v8.js

```diff
@@ -259,6 +265,19 @@ function setHeapSnapshotNearHeapLimit(limit) {
   _setHeapSnapshotNearHeapLimit(limit);
 }
 
+const detailLevelDict = {
+  __proto__: null,
+  detailed: detailLevel.DETAILED,
+  brief: detailLevel.BRIEF,
+};
+
+function getCppHeapStatistics(type = 'detailed') {
+  validateOneOf(type, 'type', ['brief', 'detailed', undefined]);
```

Comment:
The `undefined` here is unnecessary since you are specifying the default value in the argument (it will never come through as `undefined`)

Reply:
```suggestion
  validateOneOf(type, 'type', ['brief', 'detailed']);
```

Reply:
Oops 🤦🏻‍♀️